WordPress/src/main/java/org/wordpress/android/util/analytics/AnalyticsUtils.java

```diff
@@ -105,7 +104,11 @@ public static void refreshMetadata(AccountStore accountStore, SiteStore siteStor
         metadata.setNumBlogs(siteStore.getSitesCount());
         metadata.setUsername(accountStore.getAccount().getUserName());
         metadata.setEmail(accountStore.getAccount().getEmail());
-        metadata.setGutenbergEnabled(AppPrefs.isGutenbergDefaultForNewPosts());
+        // Use the first site in the list. Should be the default
```

Comment:
I know this will be accurate for users that have only one site but, feels weird to just pick the first site from the list to set the `gutenberg_enabled` Tracks metadata field. The natural thing to do would be to just pick up the currently selected site, right?

Perhaps we can use [AppPrefs.getSelectedSite()](https://github.com/wordpress-mobile/WordPress-Android/blob/18a236470a673a6f19b7dea8feb4b79330ef80d2/WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java#L550) and `mSiteStore.getSiteByLocalId()` to get the SiteModel and from there the flag?

Just to be sure we're in sync here, where will the `gutenberg_enabled` Tracks metadata flag come from in WPiOS @etoledom ?

Reply:
Right, nice call. Will change the code and get the current selected site. cc @etoledom

Reply:
Seems fixed with b66a9c52af8a46b1ba23a58960c01774ff1db4d2.

Reply:
yes, it was fixed there, left it unsolved to make sure @etoledom will see the changes.